config/owasp/suppressions.xml

```diff
@@ -32,6 +33,7 @@
     <cve>CVE-2022-46364</cve>
     <cve>CVE-2022-45688</cve>
     <cve>CVE-2023-24998</cve>
+    <cve>CVE-2022-40705</cve>
```

Comment:
This CVE appears to be a false positive - see #116.  Merging the latest changes from master should include the suppression.

Reply:
addressed